source/reference/minio-mc-admin/mc-admin-cluster-iam-import.rst

```diff
@@ -21,6 +21,33 @@ The :mc:`mc admin cluster iam import` command imports :ref:`IAM <minio-authentic
 
 You can use this command to manually restore IAM metadata settings for a MinIO deployment.
 
+.. versionadded:: mc RELEASE.2024-09-09T07-53-10Z
+
+   The command outputs the results of the import, including the following:
```

Comment:
I wonder if less is more here, because this list could change over time. e.g.

```
The command outputs results of the imports, such as:

- The number of IAM entities imported
- The number of assigned policies
- The number of IAM entities that failed to import

Reply:
This makes sense. Simplified the list.